### PR TITLE
Use DPC++ runtime from package `dpcpp_cpp_rt`

### PR DESCRIPTION
**Pull request by PokhodenkoSA**

### CODE REVIEW

**Comment by diptorupd** [comment]
Thank you for getting to it :). I wanted to do this eventually.

File: README.md [hunk 1]
```diff
@@ -20,12 +20,14 @@ Build and Install Conda Package
 conda create -n build-env conda-build
 conda activate build-env
 ```
-2. Build conda package
+2. Activate oneAPI DPC++ compiler environmnet and build conda package
 ```bash
+source ${ONEAPI_ROOT}/compiler/latest/env/vars.sh
```
Comment (diptorupd) [inline README.md:25]: I encountered few things when building this locally that we should capture in the README:

- We need IntelPython to get the `dpcpp_cpp_rt` package
- `ONEAPI_ROOT` should be defined separately if `setvars.sh` is not called.

**Review by diptorupd — APPROVED** [review]
Apart from the changes to README LGTM. Thanks!

File: README.md (hunk 1 above)
Comment (oleksandr-pavlyk) [reply README.md:25]: A comment about the assumptions that oneAPI has been activated just above that line would be very appropriate in `README.md`.

**Review by oleksandr-pavlyk — APPROVED** [review]

**Pull request merged**